src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days < 3) {
+    total = days * 40;
+  }
```

Comment:
we can reduce the number of `if`s. Set this value as default and check if we need to apply discount